x-pack/plugins/observability/public/pages/overview/old_overview_page.tsx

```diff
@@ -130,12 +138,18 @@ export function OverviewPage({ routeParams }: Props) {
                 })}
                 hasError={false}
               >
-                <AlertsTableTGrid
-                  setRefetch={setRefetch}
-                  rangeFrom={relativeTime.start}
-                  rangeTo={relativeTime.end}
-                  indexNames={indexNames}
-                />
+                <CasesContext
```

Comment:
@academo Thanks for the fix. We will make sure to create a wrapper component that contains cases context so that both alerts and overview page can share

Reply:
@mgiota  I was thinking on putting the `CasesContext` all the way up in the observability plugin. To have it available everywhere. What do you think?

Reply:
- Yep exactly that was my thought as well. This will avoid us having to write the same code in multiple places

Reply:
why not take this line of reasoning 1 step further? 😬